analyticsdataserver/settings/base.py

```diff
@@ -271,7 +271,11 @@
 
 ENABLE_ADMIN_SITE = False
 
+# base url to generate link to learner profile page
```

Comment:
This should actually be the URL to generate a link to the User API, not the HTML page.